client/mycart/src/app/shared/navbar/navbar.component.html

```diff
@@ -26,7 +26,7 @@
                 </li>
             </ul>
             <div>
-                <a class="btn btn-info me-2 text-light login-btn" [routerLink]="['/customer/customerHome']"
+                <a class="btn btn-info me-2 text-light login-btn" [routerLink]="['/login']"
```

Comment:
Where is the href attribute or routerLine?

Reply:
Gave the router Line ad/login instead of /customer/customerHome

Reply:
Done